pandas/tests/resample/test_base.py

```diff
@@ -24,6 +24,51 @@
 from pandas.core.indexes.timedeltas import timedelta_range
 from pandas.core.resample import _asfreq_compat
 
+# a fixture value can be overridden by the test parameter value. Note that the
```

Comment:
Could you just directly parameterize tests instead of using this pattern? This pattern was removed in the last release or so

Reply:
Done and removed the unused parametrizations